HTML/4. text.md

```diff
@@ -0,0 +1,93 @@
+# Text Styling
+
+You can style text in HTML in multiple different ways, you can make it bold, italicized, underlined, striketrough, superscript and subscript.
+___
+## Bold 
+You can do bold in two ways, the `<b>` tag or the `<strong>` tag, in terms of HTML semantics you should use `<strong>`
+
+Here's how it would look rendered on a webpage
+
+Code:
+
+```html
+<p>Hello <strong>how</strong> are you?</p>
+```
+
+Output:
+
+
+![Bold Text Picture](images/bold.png)
+
+**Look [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element#inline_text_semantics) for more information about HTML Semantics**
+___
+## Italics
+You can do Italics through two tags, the `<i>` tag or the `<em>` tag, and `<em>` is semantically correct.
+
+Here's how some italicized text would look on a webpage
+
+Code:
+
+```html
+<p>Hello <em>how</em> are you?</p>
+```
+
+Output:
+
+![Italicized Text Picture](images/italics.png)
+___
+## Underlined
+You can make text underlined through the `<u>` tag or the `<ins>`, but later on with CSS, there is a better way of making text underlined.
+
+Example of underlined Text
+
+Code:
+
+```html
+<p>Hello <ins>how</ins> are you?</p>
+```
+
+Output:
+
+![Underlined Text Picture](images/under.png)
+___
+## Strikethrough
+Strikethrough text can be made with three tags, the `<strike>` tag, the `<s>` tag, and the `<del>` tag. The `<del>` tag is the most semantically correct.
```

Comment:
Again I don't know what del is but I guess it is delete and I don't know what s is. If they have meanings the state them